models/task/task.go

```diff
@@ -49,20 +50,20 @@ func FindTask(listID int64, taskID int64) (*TaskStruct, error) {
 		logging.SharedInstance().MethodInfo("task", "FindTask", true).Errorf("cannot find task or list did not contain task: %v", taskID)
 		return nil, errors.New("cannot find task or list did not contain task")
 	}
-	task := NewTask(id, listId, userId, issueNumber, title, description)
+	task := NewTask(id, listId, projectId, userId, issueNumber, title, description)
 	return task, nil
 }
 
-func FindByIssueNumber(issueNumber int) (*TaskStruct, error) {
+func FindByIssueNumber(projectID int64, issueNumber int) (*TaskStruct, error) {
 	objectDB := &db.Database{}
 	var interfaceDB db.DB = objectDB
 	table := interfaceDB.Init()
 	defer table.Close()
 
-	var id, listId, userId int64
+	var id, listId, userId, projectId int64
```

Comment:
__[golint]__
* [golint] var listId should be listID
* [golint] var userId should be userID
* [golint] var projectId should be projectID